### ISSUE TITLE
Knockout SelectedOptions not set when onChange fires

### ISSUE THREAD

**Issue by HRasch** [davidstutz/bootstrap-multiselect#930]
An Erros has been introduced with issue #575. SelectedOptions are updated when this.$select.change is called in line 635

                    this.options.onChange($option, checked); => OnChange is fired
                    // Do not update select all or optgroups on select all change!
                    this.updateSelectAll();

                    if (this.options.enableClickableOptGroups && this.options.multiple) {
                        this.updateOptGroups();
                    }
                }

                this.$select.change(); => SelectedOptions Binding Updated
                this.updateButtonText();

Because onChange fires before the SelectedOptions-Binding is updated, the Observable-State is wrong.

**Comment by davidstutz** [comment]
Ok, feel free to submit a PR for this problem. I will not look into this as I am not using Knockout.

**Comment by flyfishMT** [comment]
Hello, does this mean the knockout binding is broken?  Is there an older version I can run? When I select or deselect an option it is immediately set back. Thanks

**Comment by HRasch** [comment]
Yes, it is broken. I did a PR with my solution to fix it. Please check it out.

**Comment by flyfishMT** [comment]
It still seems a little problematic.
